components-api/src/main/scala/pl/touk/nussknacker/engine/api/typed/typing.scala

```diff
@@ -57,6 +59,9 @@ object typing {
   case class TypedObjectTypingResult(fields: ListMap[String, TypingResult],
                                      objType: TypedClass,
                                      additionalInfo: Map[String, AdditionalDataValue] = Map.empty) extends SingleTypingResult {
+    override def value: Option[Map[String, Any]] =
```

Comment:
Can we preserve order of fields (ListMap). I'm not sure if someone will use it, but It was such requirement for types, for values could be the same